### PR TITLE
Don't filter out old versions or -delta from the RELEASES file. Closes #22

### LINKED ISSUE #22: Non-optimal update behavior with Squirrel.Windows
Squirrel.Windows works well with simply all releases served statically: https://github.com/Squirrel/Squirrel.Windows/issues/517

However nuts is really convenient for many different reasons, but currently, it serves a truncated 'RELEASES' file, with `-delta` files filtered out.

The result is (from what I'm seeing)

  * squirrel downloads the RELEASES file
  * squirrel downloads the full .nuget package once
  * squirrel notifies that there's a new version
  * squirrel downloads the RELEASES file again
  * squirrel downloads the full .nuget package a second time
  * squirrel unpacks the upgrade

Ideally, what would happen is this:

  * squirrel downloads the RELEASES file
  * squirrel notifies that there's a new version
  * squirrel downloads the delta .nuget package once
  * squirrel applies the delta upgrade

Since it looks like `-delta` nupkg are filtered on purpose, I was wondering what was the thinking behind it?

### PR DESCRIPTION
**Pull request by fasterthanlime**
Also fixes typo :)

Can confirm that this does wonder for [our desktop app](http://github.com/itchio/itch)

Squirrel.Windows checks the RELEASES file and happily picks the delta update that is much, much smaller than the full one.

(It still downloads it twice for some reason (presumably to get changelogs?), though, but that's okay — two times 60kb is still pretty small :shipit:)

### CODE REVIEW

**Comment by fasterthanlime** [comment]
(Also sorry if my initial comments sounded harsh/entitled, that wasn't my intention at all! :chestnut: is a great piece of software and I'm really happy you made it ^_^)

**Comment by SamyPesse** [comment]
Thanks, I'll run some tests with our application in the coming days the merge the PR :rocket:

File: lib/index.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ module.exports = function nuts(opts) {
                 if (req.useragent.isLinux64) platform = platforms.LINUX_64;
             }
 
-            if (!platform) return next(new Error('No platform specified and impossible to detect one'));
+            if (!platform) return next(new Error('We could not detect your platform.\n\nPlease pick a download directly from: https://github.com/itchio/itch/releases\n\nThanks!'));
```
Comment (SamyPesse) [inline lib/index.js]: Hum?

Comment (fasterthanlime) [inline lib/index.js]: Ha, I didn't expect the PR to be based on master. Will fix later today, ignore for now

**Comment by fasterthanlime** [comment]
By the way there is an issue with this PR that is fixed by https://github.com/itchio/nuts/commit/650d399ad1a584b0d0c430684d289b6ed5716baf

The other commit is unrelated to the PR, sorry about the noise!

**Comment by fasterthanlime** [comment]
Closed in favor of #24

**Pull request closed without merging**